sw/device/lib/dif/templates/dif_autogen.c.tpl

```diff
@@ -0,0 +1,211 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is auto-generated.
+
+#include "sw/device/lib/dif/dif_${ip_snake}_autogen.h"
+
+#include "${ip_snake}_regs.h"  // Generated.
```

Comment:
There's gonna be a lot of inputs to these files... Maybe worth documenting in the templates themselves using the `<%doc>` tag?

Reply:
Good thinking. Fixed.